kmod/core/core.c

```diff
@@ -1008,9 +1008,22 @@ int kpatch_register(struct kpatch_module *kpmod, bool replace)
 		func->op = KPATCH_OP_NONE;
 	} while_for_each_linked_func();
 
+/* HAS_KERNEL_TAINT - upstream 2992ef29ae01 "livepatch/module: make TAINT_LIVEPATCH module-specific" */
+#ifdef RHEL_RELEASE_CODE
+# if RHEL_RELEASE_CODE >= RHEL_RELEASE_VERSION(7, 4)
+#  define HAS_KERNEL_TAINT
```

Comment:
Not to be picky, but can you change this to HAS_MODULE_TAINT instead? Since with HAS_KERNEL_TAINT I don't immediately understand that we're actually talking about the new per-module taint referenced in that commit.